util/eventbus/bus.go

```diff
@@ -86,6 +103,11 @@ func (b *Bus) pump(ctx context.Context) {
 		}
 		return b.write
 	}
+	acceptPublish := func(val publishedEvent) {
+		val.Accepted = time.Now()
```

Comment:
Oh, I see, accepted is notionally _before_ published, in that it represents when we received it from the host.

Reply:
Other way around, confusingly. Published is "the publisher wants to publish this", Accepted is "the bus has picked up the parcel and is working on it". This clearly shows the words aren't well picked :)